pkg/preflights/host-preflight.yaml

```diff
@@ -834,3 +835,18 @@ spec:
           - pass:
               when: "a-subnet-is-available"
               message: Specified CIDR is available.
+    - sysctl:
+        checkName: Sysctl Kernel Parameters
+        outcomes:
+          - fail:
+              when: 'net.ipv4.conf.default.arp_filter > 0'
```

Comment:
If someone uses a network interface other than default, does this value need to use that interface instead?

Reply:
So the `default` entry doesn't really translate into a particular interface but is used instead whenever a new interface is created, that's the value that gets set, which is exactly what happens when `calico` spins up its specific interfaces (and that's what poses a problem).